src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/SemanticTokensTest.cs

```diff
@@ -137,53 +136,51 @@ public class MyClass
             }
             """;
 
-        await VerifySemanticTokensAsync(documentText, precise, isRazorFile: true, supportsVSExtensions: supportsVSExtensions);
+        await VerifySemanticTokensAsync(documentText, isRazorFile: true, supportsVSExtensions: supportsVSExtensions);
     }
 
     [Theory]
     [CombinatorialData]
-    public async Task GetSemanticTokens_CSharp_Implicit(bool serverSupportsPreciseRanges, bool precise, bool supportsVSExtensions)
+    public async Task GetSemanticTokens_CSharp_Implicit(bool supportsVSExtensions)
     {
         var documentText = """
             @addTagHelper *, TestAssembly
             @{ var d = "txt";}
             @d
             """;
 
-        var csharpTokens = await GetCSharpSemanticTokensResponseAsync(documentText, precise, supportsVSExtensions: supportsVSExtensions);
-        await AssertSemanticTokensAsync(documentText, precise, supportsVSExtensions: supportsVSExtensions, csharpTokens: csharpTokens, serverSupportsPreciseRanges: serverSupportsPreciseRanges);
-        VerifyTimesLanguageServerCalled(serverSupportsPreciseRanges, precise);
+        var csharpTokens = await GetCSharpSemanticTokensResponseAsync(documentText, supportsVSExtensions: supportsVSExtensions);
+        await AssertSemanticTokensAsync(documentText, supportsVSExtensions: supportsVSExtensions, csharpTokens: csharpTokens);
     }
 
     [Theory]
     [CombinatorialData]
-    public async Task GetSemanticTokens_CSharp_VersionMismatch(bool serverSupportsPreciseRanges, bool precise, bool supportsVSExtensions)
+    public async Task GetSemanticTokens_CSharp_VersionMismatch(bool supportsVSExtensions)
     {
         var documentText = """
             @addTagHelper *, TestAssembly
             @{ var d = }
             """;
 
-        var csharpTokens = await GetCSharpSemanticTokensResponseAsync(documentText, precise, supportsVSExtensions: supportsVSExtensions);
-        await AssertSemanticTokensAsync(documentText, precise, csharpTokens: csharpTokens, documentVersion: 21, serverSupportsPreciseRanges: serverSupportsPreciseRanges);
-        VerifyTimesLanguageServerCalled(serverSupportsPreciseRanges, precise);
+        var csharpTokens = await GetCSharpSemanticTokensResponseAsync(documentText, supportsVSExtensions: supportsVSExtensions);
```

Comment:
Now that the other verification is gone, can this test, and the one above, now just call `VerifySemanticTokensAsync` like the rest of the tests?

Reply:
Evidently not without changing the baseline, so I'll punt on this unless you feel strongly.

Reply:
Nerd sniped myself and had a look why, and my suggestion was wrong.